File: VideoEidtexter/src/main/java/com/cgfay/cameralibrary/widget/sticker/DeleteIconEvent.java
package com.cgfay.cameralibrary.widget.sticker;

import android.view.MotionEvent;

/**
 * @author wupanjie
 */

public class DeleteIconEvent implements StickerIconEvent {
  @Override public void onActionDown(StickerView stickerView, MotionEvent event) {

  }

  @Override public void onActionMove(StickerView stickerView, MotionEvent event) {

  }

  @Override public void onActionUp(StickerView stickerView, MotionEvent event) {
    stickerView.removeCurrentSticker();
  }
}

File: videov2/src/main/java/com/owoh/video/widget/sticker/DeleteIconEvent.kt
package com.owoh.video.widget.sticker

import android.view.MotionEvent

/**
 * @author wupanjie
 */

class DeleteIconEvent : StickerIconEvent {
    override fun onActionDown(stickerView: StickerView, event: MotionEvent) {

    }

    override fun onActionMove(stickerView: StickerView, event: MotionEvent) {

    }

    override fun onActionUp(stickerView: StickerView, event: MotionEvent) {
        stickerView.removeCurrentSticker()
    }
}
